### ISSUE TITLE
Make a temporary home and a future plan for presidential matching fund submissions

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#3251]
### Summary

**What we are after:**
We need a home for the matching funds submissions, both historical and future. Right now they are housed at https://classic.fec.gov/finance/disclosure/other_disclosure.shtml and linked off our public funding information page for citizens at https://www.fec.gov/introduction-campaign-finance/understanding-ways-support-federal-candidates/presidential-elections/public-funding-presidential-elections/#reports-and-resources (under the heading "other presidential campaign finance data.")

### Background
Usually the matching fund submissions come from Audit to the Press Office who coordinates their posting, and to Public Records Office, which handles the website posting

### Completion criteria

- [x] @dorothyyeager will create a transition landing page for the matching fund submissions.

- [x] @dorothyyeager will create a transition page for 2020 matching fund submissions

- [x] @dorothyyeager will  adjust redirects for matching funds pages in transition and www redirects:  
https://transition.fec.gov/finance/disclosure/2016MatchingFundSubmissions.shtml, https://transition.fec.gov/finance/2012matching/2012matching.shtml, https://transition.fec.gov/finance/2008matching/2008matching.shtml, https://transition.fec.gov/finance/disclosure/2004MatchingFundSubmissions.shtml

- [x] @patphongs will move these pages from classic to transition:
2016 Presidential Campaign Matching Fund Submissions : https://classic.fec.gov/finance/disclosure/2016MatchingFundSubmissions.shtml
2012 Presidential Campaign Matching Fund Submissions : https://classic.fec.gov/finance/2012matching/2012matching.shtml
2008 Presidential Campaign Matching Fund Submissions: https://classic.fec.gov/finance/2008matching/2008matching.shtml
2004 Presidential Campaign Matching Fund Submissions: https://classic.fec.gov/finance/disclosure/2004MatchingFundSubmissions.shtml

- [x] Link the pages listed above on the new transition page for matching fund submissions (Note: 2020 page should not go live until FEC is ready to go live.)

- [x] Test that redirects for the 2004-2016 pages are working.

- [x] Add a link to the new transition page to the "matching funds" part of  https://www.fec.gov/introduction-campaign-finance/understanding-ways-support-federal-candidates/presidential-elections/public-funding-presidential-elections/ 

- [x] Once 2020 page is ready to go live, remove the exclusion in the transition-stage S3 script, `sync-prod.sh`, by removing this line:
`--exclude 'finance/disclosure/2020presidentialmatchingfundsubmissions.shtml'` -- ***This has been moved into its own ticket at #3385 ***

- [x] Make a plan to make a more permanent home in Wagtail for presidential matching fund submissions  - Done in #3386

### Future work
- Remove exclusion for 2020 page (#3385)
- Put in a redirect for the original landing page on classic

**Comment by patphongs** [comment]
@dorothyyeager These pages are ready to go on transition:
2016 Presidential Campaign Matching Fund Submissions : https://transition.fec.gov/finance/disclosure/2016MatchingFundSubmissions.shtml
2012 Presidential Campaign Matching Fund Submissions : https://transition.fec.gov/finance/2012matching/2012matching.shtml
2008 Presidential Campaign Matching Fund Submissions: https://transition.fec.gov/finance/2008matching/2008matching.shtml
2004 Presidential Campaign Matching Fund Submissions: https://transition.fec.gov/finance/disclosure/2004MatchingFundSubmissions.shtml

**Comment by dorothyyeager** [comment]
I've created the pages. Will show them to @AmyKort and @patphongs before publishing and putting in redirects. I also updated an outdated reference on the 2016 page to 2 U.S.C. (now 52 U.S.C.)

**Comment by dorothyyeager** [comment]
Amy's approved publishing them. Next step is to put in the redirects.

**Comment by AmyKort** [comment]
Thanks @patphongs and @dorothyyeager !

**Comment by dorothyyeager** [comment]
Showed pages to Jason in Public Records and he's on board with them. The 2020 page should not go live until they are ready to go live with the first submission. @johnnyporkchops is helping me by taking the 2020 page down, excluding it from future syncs, and then we'll put it back up. I'll begin the redirects when he gives me a heads up it's OK to do that.

For the landing page, we are labeling 2020 as "Coming soon" and removing the link to the new page. The new pages will be maintained by Public Records. cc @AmyKort @patphongs

**Comment by johnnyporkchops** [comment]
I added a checklist item above to remove the exclusion for the 2020 page once it is ready to go live.

**Comment by dorothyyeager** [comment]
Put in redirects for the four classic pages to the new transition pages; they're ready for @patphongs to merge and deploy. Once they're good, I'll add the link to the new page.

The 2020 page may not go live for some time. I'll move the future work with that into another ticket.

**Comment by dorothyyeager** [comment]
The transition pages are up and running. The link to their landing page (https://transition.fec.gov/finance/disclosure/presidential_matching_fund_submissions_2004_present.shtml) has been added at https://www.fec.gov/introduction-campaign-finance/understanding-ways-support-federal-candidates/presidential-elections/public-funding-presidential-elections/#reports-and-resources. We'll need to redirect the original landing page so I'll put in a PR for that. The next step for upgrading from this MVP would be design work, so I've put in a ticket for that at #3386.

When the 2020 page is ready to go live, we'll need to remove an exclusion. See #3385. I'll make sure to let Public Records know that they should keep @johnnyporkchops in the loop for when the page goes live.

Closing this issue as the pages are done and we're ready to turn them over to Public Records and Press for updating when needed.
